app/pages/resource/resource-info/ResourceInfo.js

```diff
@@ -71,12 +94,94 @@ function ResourceInfo({
       </ResourcePanel>
+
+      {accessibilityInformation && (
+        <ResourcePanel
+          className="app-ResourceInfo__accessibility-resource-panel"
+          componentClass="div"
+          defaultExpanded={false}
+          header={(
+            <div className="app-ResourceInfo__accessibility-resource-panel__title-container">
+              <h3>{t('ResourceInfo.accessibilityTitle')}</h3>
+              <div className="app-ResourceInfo__accessibility-resource-panel__title-container__shortcomings">
+                <img
+                  alt=""
+                  className="app-ResourceInfo__accessibility__icon"
+                  src={
+                    nAccessibilityShortcomings === 0
+                      ? iconAccessibilityOk
+                      : iconAccessibilityError
+                  }
+                />
+                {nAccessibilityShortcomings === 0
+                  ? t('ResourceInfo.noAccessibilityShortcomings')
+                  : t('ResourceInfo.foundAccessibilityShortcomings', {
+                    nShortcomings:
+                    nAccessibilityShortcomings,
+                  })}
+              </div>
+            </div>
+)}
+        >
+          <div className="app-ResourceInfo__accessibility">
+            {accessibilityInformation.details.length > 0 && (
+              <div>
+                <h4>{t('ResourceInfo.accessibilityDetailsTitle')}</h4>
+                {accessibilityInformation.details.map(
+                  (sentenceGroup, sentenceGroupId) => (
+                    <React.Fragment
+                      key={`accessibility-sentence-group-${sentenceGroupId}`}
+                    >
+                      <b>{sentenceGroup.sentenceGroup}</b>
+                      <ul>
+                        {sentenceGroup.sentences.map((sentence, sentenceId) => (
+                          <li key={`sentence-${sentenceId}`}>{sentence}</li>
+                        ))}
+                      </ul>
+                    </React.Fragment>
+                  ),
+                )}
+              </div>
+            )}
+
+            {accessibilityInformation.shortcomings.length > 0 && (
+              <div>
+                <h4>{t('ResourceInfo.accessibilityShortcomingsTitle')}</h4>
+                {accessibilityInformation.shortcomings.map(
+                  (sentenceGroup, sentenceGroupId) => (
+                    <React.Fragment
+                      key={`accessibility-sentence-group-${sentenceGroupId}`}
+                    >
+                      <div className="app-ResourceInfo__accessibility__shortcoming-sentence-group">
+                        <img
+                          alt="accessibility-shortcoming"
+                          className="app-ResourceInfo__accessibility__icon"
+                          src={iconAccessibilityError}
+                        />
+                        <b>{sentenceGroup.sentenceGroup}</b>
```

Comment:
`strong` semantically emphasizes the respective text as important. 
```suggestion
                        <strong>{sentenceGroup.sentenceGroup}</strong>
```

Reply:
Done.